siteapp/views_landing.py

```diff
@@ -46,7 +48,7 @@ def homepage(request):
     signup_form.fields['username'].widget.attrs.pop("autofocus", None)
     login_form.fields['login'].widget.attrs.pop("autofocus", None)
 
-    if "signup" in request.path or request.POST.get("action") == "signup":
+    if SIGNUP in request.path or request.POST.get("action") == SIGNUP:
```

Comment:
Just a note: In the future we may want to have a `constants.py` file that contains all of the constants for the application, so that all constants are in one place for referencing throughout the codebase.